src/brain/issueLabelHandler/followups.ts

```diff
@@ -3,7 +3,43 @@ import * as Sentry from '@sentry/node';
 
 import { isFromABot } from '@utils/isFromABot';
 
-const REPOS_TO_TRACK_FOR_TRIAGE = new Set(['sentry', 'sentry-docs']);
+const REPOS_TO_TRACK_FOR_FOLLOWUPS = new Set([
+  'sentry',
+  'sentry-docs',
+  'arroyo',
+  'cdc',
+  'craft',
+  'relay',
+  'responses',
+  'self-hosted',
+  'sentry-native',
+  'snuba',
+  'snuba-sdk',
+  'symbolic',
+  'symbolicator',
+  'test-ttt-simple',
+  'wal2json',
+
+  // Web team, T1
+  'sentry-javascript',
+  'sentry-python',
+  'sentry-php',
+  'sentry-laravel',
+  'sentry-symfony',
+  'sentry-ruby',
+
+  // Mobile team, T1
+  // https://www.notion.so/sentry/346452f21e7947b4bf515d5f3a4d497d?v=cad7f04cf9064e7483ab426a26d3923a
+  'sentry-cocoa',
+  'sentry-java',
+  'sentry-react-native',
+  'sentry-unity',
+  'sentry-dart',
+  'sentry-android-gradle-plugin',
+  'sentry-dotnet',
+  'sentry-dart-plugin',
+  'test-sentry-app',
+]);
```

Comment:
If this duplicates another set, should we have one constant that is imported in both places?

Reply:
yes we should :)